dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/MinMaxDataElementController.java

```diff
@@ -198,4 +215,42 @@ private MinMaxDataElement getReferences(MinMaxDataElement m) throws WebMessageEx
           notFound("Invalid required parameters: source, dataElement, optionCombo"));
     }
   }
+
+  // Bulk import
+  @PostMapping(value = "/values", consumes = "application/json")
+  @RequiresAuthority(anyOf = F_MINMAX_DATAELEMENT_ADD)
+  @ResponseStatus(HttpStatus.NO_CONTENT)
+  public void bulkPostJson(
+      @RequestBody MinMaxValueBatchRequest request,
+      @RequestParam(value = "importStrategy", defaultValue = "UPDATE") String importStrategy)
+      throws BadRequestException {
+
+    try {
```

Comment:
If the service directly throws the `BadRequestException` this try-catch business will also disappear nicely in the controller ;)